### PR TITLE
Pods 9712

### PR DESCRIPTION
**Pull request by PavelVjalicin**

### CODE REVIEW

File: app/views/EmailView.scala.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+govukInput: GovukInput,
+govukButton: GovukButton
+)
+@(form: Form[_], schemeName: String, entityName: String, entityType: String, paragraphs: Seq[String], submitCall: Call)(implicit request: Request[_], messages: Messages)
+
+@field = @{form("value")}
+
+@layout(messages("messages__enterEmail_pageHeading", entityName)) {
+<div>
+    @* @if(paragraphs) {
```
Comment (PavelVjalicin) [inline app/views/EmailView.scala.html:30]: Commented out code.

File: app/views/HasReferenceValueView.scala.html
Comment (PavelVjalicin) [inline app/views/HasReferenceValueView.scala.html:1]: Empty file? Need to remove?

File: test/controllers/adviser/EnterEmailControllerSpec.scala [hunk 2]
```diff
@@ -113,11 +105,10 @@ class EnterEmailControllerSpec extends ControllerSpecBase
           .onPageLoad(NormalMode)(fakeDataRequest(userAnswers))
 
       status(result) mustBe OK
-      verify(mockRenderer, times(1))
-        .render(templateCaptor.capture(), jsonCaptor.capture())(any())
-      templateCaptor.getValue mustEqual templateToBeRendered
-      val json: JsObject = Json.obj("form" -> form.fill(formData))
-      jsonCaptor.getValue must containJson(commonJson ++ json)
+
+      contentAsString(result) must include(messages("messages__enterEmail_pageHeading", advisorName))
```
Comment (PavelVjalicin) [inline test/controllers/adviser/EnterEmailControllerSpec.scala:109]: Rewrite this to use compare view function.

File: test/controllers/establishers/company/contact/EnterEmailControllerSpec.scala [hunk 3]
```diff
@@ -114,11 +106,11 @@ class EnterEmailControllerSpec extends ControllerSpecBase
...
       status(result) mustBe OK
-      verify(mockRenderer, times(1))
-        .render(templateCaptor.capture(), jsonCaptor.capture())(any())
-      templateCaptor.getValue mustEqual templateToBeRendered
-      val json: JsObject = Json.obj("form" -> form.fill(formData))
-      jsonCaptor.getValue must containJson(commonJson ++ json)
+
+      contentAsString(result) must include (messages("messages__enterEmail_pageHeading",
+        company.companyName))
+      contentAsString(result) must include (formData)
```
Comment (PavelVjalicin) [inline test/controllers/establishers/company/contact/EnterEmailControllerSpec.scala:112]: should use compare function

**Comment by PavelVjalicin** [comment]
Remove .njx files that were migrated.
Get a list of URL to test for this to testers and put the ticket into testing

File: app/views/HasReferenceValueView.scala.html
Comment (nonebula) [reply app/views/HasReferenceValueView.scala.html:1]: Removed.

File: app/views/EmailView.scala.html (hunk 1 above)
Comment (nonebula) [reply app/views/EmailView.scala.html:30]: Uncommented.

File: test/controllers/establishers/company/director/details/DirectorEnterUTRControllerSpec.scala [hunk 4]
```diff
@@ -55,26 +56,25 @@ class DirectorEnterUTRControllerSpec
...
+//      "pageHeading"     -> "What is Jane Doe’s UTR?",
+//      "schemeName"    -> "Test scheme name",
+//      "legendClass"   -> "govuk-visually-hidden",
+//      "paragraphs"    -> Json.arr(
+//        "This is a 10-digit or 13-digit number. It may also start or end with the letter ‘k’.",
+//        "You can find it on tax returns and other documents from HMRC. It might be called ‘reference’, ‘UTR’ or ‘official use’."
+//      ),
+//      "isPageHeading" -> true
+//    )
+
```
Comment (jamdade-sharad) [inline test/controllers/establishers/company/director/details/DirectorEnterUTRControllerSpec.scala:72]: remove commented code if not required

File: test/controllers/establishers/individual/details/EstablisherEnterNINOControllerSpec.scala [hunk 5]
```diff
@@ -55,16 +56,15 @@ class EstablisherEnterNINOControllerSpec
...
-    )
+
+//  private val commonJson: JsObject =
+//    Json.obj(
+//      "pageTitle"     -> "What is the individual’s National Insurance number?",
+//      "pageHeading"     -> "What is Jane Doe’s National Insurance number?",
+//      "schemeName"    -> "Test scheme name",
+//      "legendClass"   -> "govuk-label--xl",
+//      "isPageHeading" -> true
+//    )
```
Comment (jamdade-sharad) [inline test/controllers/establishers/individual/details/EstablisherEnterNINOControllerSpec.scala:67]: Remove unused code

File: test/controllers/establishers/partnership/details/UTRControllerSpec.scala [hunk 6]
```diff
@@ -45,68 +46,64 @@ class UTRControllerSpec extends ControllerSpecBase with NunjucksSupport with Jso
...
-    )
+//  private val commonJson: JsObject =
+//    Json.obj(
+//      "pageTitle"     -> messages("messages__enterUTR", messages("messages__partnership")),
+//      "pageHeading"     -> messages("messages__enterUTR", partnershipDetails.partnershipName),
+//      "schemeName"    -> schemeName,
+//      "paragraphs"      -> Json.arr(messages("messages__UTR__p1"), messages("messages__UTR__p2")),
+//      "legendClass"   -> "govuk-visually-hidden",
+//      "isPageHeading" -> true
+//    )
```
Comment (jamdade-sharad) [inline test/controllers/establishers/partnership/details/UTRControllerSpec.scala:61]: Remove unused code

File: test/controllers/establishers/partnership/details/VATControllerSpec.scala [hunk 7]
```diff
@@ -44,69 +46,63 @@ class VATControllerSpec extends ControllerSpecBase with NunjucksSupport with Jso
...
-    )
+//  private val commonJson: JsObject =
+//    Json.obj(
+//      "pageTitle"     -> messages("messages__vat", messages("messages__partnership")),
+//      "pageHeading"     -> messages("messages__vat", partnershipDetails.partnershipName),
+//      "schemeName"    -> schemeName,
+//      "paragraphs"      -> Json.arr(messages("messages__vat__p", partnershipDetails.partnershipName)),
+//      "legendClass"   -> "govuk-visually-hidden",
+//      "isPageHeading" -> true
+//    )
```
Comment (jamdade-sharad) [inline test/controllers/establishers/partnership/details/VATControllerSpec.scala:60]: remove commented code

File: test/controllers/trustees/company/details/UTRControllerSpec.scala [hunk 8]
```diff
@@ -36,76 +36,73 @@ import services.common.details.CommonEnterReferenceValueService
...
-    )
+//  private val commonJson: JsObject =
+//    Json.obj(
+//      "pageTitle"     -> messages("messages__enterUTR", messages("messages__company")),
+//      "pageHeading"     -> messages("messages__enterUTR", companyDetails.companyName),
+//      "schemeName"    -> schemeName,
+//      "paragraphs"      -> Json.arr(messages("messages__UTR__p1"), messages("messages__UTR__p2")),
+//      "legendClass"   -> "govuk-visually-hidden",
+//      "isPageHeading" -> true
+//    )
```
Comment (jamdade-sharad) [inline test/controllers/trustees/company/details/UTRControllerSpec.scala:58]: remove commented code

File: test/controllers/trustees/company/details/VATControllerSpec.scala [hunk 9]
```diff
@@ -36,75 +36,71 @@ import services.common.details.CommonEnterReferenceValueService
...
+
+//  private val commonJson: JsObject =
+//    Json.obj(
+//      "pageTitle"     -> messages("messages__vat", messages("messages__company")),
+//      "pageHeading"     -> messages("messages__vat", companyDetails.companyName),
+//      "schemeName"    -> schemeName,
+//      "paragraphs"      -> Json.arr(messages("messages__vat__p", companyDetails.companyName)),
+//      "legendClass"   -> "govuk-visually-hidden",
+//      "isPageHeading" -> true
+//    )
```
Comment (jamdade-sharad) [inline test/controllers/trustees/company/details/VATControllerSpec.scala:59]: remove commented code

**Review by jamdade-sharad — CHANGES_REQUESTED** [review]
Remove commented code

**Review by jamdade-sharad — APPROVED** [review]

**Pull request merged**